app/assets/javascripts/custom.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Created by mkaba on 10/12/17.
+ */
+    $('.close-modal').click( function () {
+        $('#gallery').css('display', 'none');
```

Comment:
use `.hide()`

Reply:
i think i dont need this code, i use this in previous version. Now i use bootstrap class '.close' for this action, i will remove it

Reply:
first time i made custom modal window

Reply:
and i forgot to delete it